packages/web-components/src/pane/pane.template.ts

```diff
@@ -0,0 +1,39 @@
+import { ElementViewTemplate, html, ref } from '@microsoft/fast-element';
+import type { Pane } from './pane.js';
+
+/**
+ * The template for the Pane component.
+ * @public
+ */
+
+export function paneTemplate<T extends Pane>(): ElementViewTemplate<T> {
+  return html<T>`
+    <template
+      role="${x => (x.modal ? 'dialog' : 'complementary')}"
+      ?open="${x => x.open}"
+      ?modal="${x => x.modal}"
```

Comment:
Why does modal need to be boolean if we are setting aria-modal? One has semantics here and the other would just be a visual hook.